conda.recipe/bld.bat

```diff
@@ -0,0 +1,3 @@
+:: Use cython coverage on build
+"%PYTHON%" setup.py build_ext -i --compiler=msvc
```

Comment:
This command does not install nothing at all to ``site-packages``, it only compiles all files locally